lib/v2/pkuszcs/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/xwzx/:type', require('./xwzx'));
```

Comment:
Please use the namespace `pku` instead of `pkuszcs` since that's the right domain name